src/Checks/VersionCheck.php

```diff
@@ -43,6 +47,17 @@ private function getMysqlVersion(): string
         return app()->runningUnitTests() ? '8.0.0' : DB::select('select version()')[0]->{'version()'};
     }
 
+    private function getNodeVersion(): string
+    {
+        $fileContents = File::get(base_path('package.json'));
+        if (Str::length($fileContents) === 0) {
+            return 'N/A';
+        }
+        $data = json_decode($fileContents, true);
+
+        return Arr::exists($data, 'engines') && Arr::exists($data['engines'], 'node') ? preg_replace('/[>=]/', '', $data['engines']['node']) : 'N/A';
```

Comment:
Could even be good to leave in the `>=` if it is there, tells us what the node requirements are.